uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
+  </header>
+  <div class="table-responsive">
+    <table class="table">
+      <tr>
```

Comment:
I did TH at first, but it will require extra class+CSS to be added as by default TH have grey background